assets/js/screens/Collection/List.jsx

```diff
@@ -20,13 +30,41 @@ const ScreensCollectionList = () => {
     }
   }, [data]);
 
+  const [deleteCollection] = useMutation(DELETE_COLLECTION, {
+    onCompleted({ deleteCollection }) {
+      addToast(`Collection ${deleteCollection.name} deleted successfully`, {
+        appearance: "success",
+        autoDismiss: true
+      });
+      history.push("/collection/list");
```

Comment:
Since we're already on this route in the application, we can remove the `history.push(...)` code to change the route.

Reply:
And if not using elsewhere in the component, can also remove the `import` above.

Reply:
Ok, this is deleted! Can I re-assign this PR to you?